engine/src/cmd/collide2/Ice/IcePoint.h

```diff
@@ -212,7 +212,7 @@
 									if(y<min)	y=min;
 			                                                    else if(y>max)	y=max;
 									if(z<min)	z=min;
-			                                                     elseif(z>max)	z=max;
+			                                                     else if(z>max)	z=max;
```

Comment:
This is fixed in my PR too, so it'll cause a conflict to whoever merges second. I won't block on this.

Reply:
@nabaco since this is focused, can we merge this to master then update your PR?

Reply:
Sure, the effort to rebase my PR will be minimal.